docs/README_RECON.md

```diff
@@ -104,15 +136,36 @@ class Table:
 
 ### jdbc_reader_options
 
-```python
+<table>
+<tr>
+<th>python</th>
+<th>json</th>
+</tr>
+<tr>
+<td>
+<pre lang="python">
 @dataclass
 class JdbcReaderOptions:
     number_partitions: int
     partition_column: str
     lower_bound: str
     upper_bound: str
     fetch_size: int = 100
-```
+</pre>
+</td>
+<td>
+<pre lang="json">
+"jdbc_reader_options":{
+  "number_partitions": 10,
+  "partition_column": "p_id",
```

Comment:
Please update all the actual values with <TO-BE-FILLED> elements.

Reply:
Fixed